src/main/scala/sigmastate/Values.scala

```diff
@@ -136,6 +136,9 @@ object Values {
       case Constant(value: Int, SInt) => Some(value)
       case _ => None
     }
+
+    def zero = IntConstant(0)
```

Comment:
@kushti it is better to name constants with uppercase because 1) then they can be used in pattern matching 2) they will look like objects which are semantically close